code/modules/research/machinery/part_fabricator/part_fabricator.dm

```diff
@@ -0,0 +1,521 @@
+/// The Production Producer
+/obj/machinery/part_fabricator
+	name = "experimental part fabricator"
+	desc = "A strange machine that condenses materials into advanced parts."
+	icon = 'icons/obj/machines/part_fabricator.dmi'
+	icon_state = "default"
+	circuit = /obj/item/circuitboard/machine/part_fabricator
+	resistance_flags = INDESTRUCTIBLE // dont want it to be destroyed by radballs
+	
+	pixel_x = -32
+	
+	use_power = IDLE_POWER_USE
+	idle_power_usage = 5000
+	active_power_usage = 10000000 // 10MW
```

Comment:
wouldn't be surprised if this just broke the apc code

Reply:
what exactly is wrong here? this is how these vars are meant to be used